azext_edge/edge/providers/orchestration/resources/connector/opcua/certs.py

```diff
@@ -571,8 +573,15 @@ def _upload_to_key_vault(self, secret_name: str, file_path: str, cert_extension:
             else:
                 content_type = "application/x-pem-file"
 
+            parameters = {
+                "value": content,
+                "contentType": content_type,
+                "tags": {"file-encoding": "hex"},
+            }
             return self.keyvault_client.set_secret(
-                name=secret_name, value=content, content_type=content_type, tags={"file-encoding": "hex"}
+                vault_base_url=KEYVAULT_URL.format(keyvaultName=keyvault_name),
+                secret_name=secret_name,
+                parameters=parameters,
             )
 
```

Comment:
Github doesn't let me comment on the line in question, so putting the comment here instead. The method `_update_client_secret_to_extension` has a parameter `applicationCert` in camel case, can we change it to snake case like `application_cert`?

![image](https://github.com/user-attachments/assets/2ff70d14-9fda-4692-a46c-2720450e4c29)

Reply:
of course! sorry missed this😢